src/EditorFeatures/VisualBasicTest/CodeActions/AddAwait/AddAwaitTests.vb

```diff
@@ -140,17 +140,7 @@ Module Program
 End Module
 </File>
 
-            Dim expected =
-<File>
-Imports System.Threading.Tasks
-Module Program
-    Async Function GetNumberAsync() As Task(Of Integer)
-        Dim x = (Await GetNumberAsync()).ToString()
-    End Function
-End Module
-</File>
-
-            Await TestAsync(markup, expected)
+            Await TestMissingAsync(markup)
```

Comment:
This is a regression :confused:

Reply:
I'll think more about this, but I think it's fine. I've seen `await` used in assignments and in parameters, but rarely nested this way in a chain of invocations.

Reply:
I agree with @jcouv - we can add this back if we get customer feedback, but doesn't seem an important scenario.

Reply:
@mavasani I've already updated the PR from discussion with @Neme12: if `x()$$.y()` is happy (binds and has a non-error type), then we shouldn't offer the refactoring.